tests/components/tplink/test_button.py

```diff
@@ -28,14 +33,35 @@
 def mocked_feature_button() -> Feature:
     """Return mocked tplink binary sensor feature."""
     return _mocked_feature(
-        False,
         "test_alarm",
+        value=False,
```

Comment:
python-kasa currently reports this as `"<Action>"`: https://github.com/python-kasa/python-kasa/blob/4df5fbc0dd8d8434da02431110e45eb5ad30b091/kasa/feature.py#L206